docs/content/usage.md

```diff
@@ -45,9 +45,9 @@ Now let's say you just bought `example.com` and you want to be able to send and
 We will later dig into DKIM, DMARC & SPF, but for now, these are the records that suffice in getting you up and running. Here is a short explanation of what the records do:
 
 - The **MX record** tells everyone which (DNS) name is responsible for e-mails on your domain.
-    Because you want to keep the option of running another service on the domain name itself, you run your mail server on `mail.example.com`.  
-    This does not imply your e-mails will look like `test@mail.example.com`, the DNS name of your mail server is decoupled of the domain it serves e-mails for.  
-    In theory, you mail server could even serve e-mails for `test@some-other-domain.com`, if the MX record for `some-other-domain.com` points to `mail.example.com`.  
+    Because you want to keep the option of running another service on the domain name itself, you run your mail server on `mail.example.com`.
+    This does not imply your e-mails will look like `test@mail.example.com`, the DNS name of your mail server is decoupled of the domain it serves e-mails for.
+    In theory, you mail server could even serve e-mails for `test@some-other-domain.com`, if the MX record for `some-other-domain.com` points to `mail.example.com`.
```

Comment:
I thought the two spaces at the end would have helped formatting but nope it still looked pretty bad on the rendered page :sweat_smile: 

You can see in [this preview](https://pullrequest-3235--dms-doc-previews.netlify.app/usage/#minimal-dns-setup) that the two spaces as per markdown syntax helped establish new lines properly, whereas the removal has collapsed them.

I'd still have preferred some more white-space between the lines, but the only easy way I know of that is using a list, or maybe some blank lines inbetween would do it.

---

Also the last line has a typo "you mail server" instead of "your mail server".

Reply:
Now I understand what these spaces were meant for. I use VS Code, and because it respects `.editorconfig`, it will trim whitespaces (on save).

We should adjust this with another PR.

Reply:
> We should adjust this with another PR and make this text render nicely (please no whitespaces at the end though).

That's the only way to get a new line without introducing a blank line inbetween. But as noted, the typography from our docs theme doesn't render that well, you'd need blank lines probably.

---

> I actually quite like the way it looks now :joy:

That's ok, I'm not too fussy about it.

I have a bigger issue with the themes headings blending in with regular text too much by around the third level, and not standing out as much as headings in the larger sizes :man_shrugging: 

I'm not interested in fiddling with that any time soon, and what I consider good document flow for readability may be biased anyway :sweat_smile: